bfdd/bfdd_cli.c

```diff
@@ -468,17 +491,52 @@ DEFPY_YANG(
 	return nb_cli_apply_changes(vty, NULL);
 }
 
-void bfd_cli_show_echo_interval(struct vty *vty, struct lyd_node *dnode,
-				bool show_defaults)
+void bfd_cli_show_desired_echo_transmission_interval(struct vty *vty,
+	struct lyd_node *dnode, bool show_defaults)
 {
 	uint32_t value;
 
 	if (show_defaults)
-		vty_out(vty, "  echo-interval %d\n",
-			BFD_DEF_REQ_MIN_ECHO);
+		vty_out(vty, "  echo transmit-interval %d\n",
+			BFD_DEF_DES_MIN_ECHO_TX);
 	else {
 		value = yang_dnode_get_uint32(dnode, NULL);
-		vty_out(vty, "  echo-interval %u\n", value / 1000);
+		vty_out(vty, "  echo transmit-interval %u\n", value / 1000);
+	}
+}
+
+DEFPY_YANG(
+	bfd_peer_echo_receive_interval, bfd_peer_echo_receive_interval_cmd,
+	"echo receive-interval (0-60000)$interval",
```

Comment:
Please don't make it possible to configure values under 10ms.

Suggestion: make `0` a special value named `disabled` and allow `10-60000` or `disabled`. This will make it clearer for the user configuring this feature.